api/src/functions/encryptTaxId/app.ts

```diff
@@ -13,7 +13,7 @@ export default async function handler(): Promise<void> {
   const USERS_TABLE = process.env.USERS_TABLE || "users-table-local";
   const DYNAMO_OFFLINE_PORT = process.env.DYNAMO_PORT || 8000;
   const STAGE = process.env.STAGE || "local";
-  const logger = LogWriter(`NavigatorWebService/${STAGE}`, "DataMigrationLogs");
+  const logger = LogWriter(`aws/${STAGE}`, "DataMigrationLogs");
```

Comment:
We should avoid using aws/${STAGE} as the log stream prefix, as it may create confusion by implying that the logs are AWS-managed or system logs.

If NavigatorWebService references Boomi, we can decide on a more descriptive service name cc @jphechter 

To maintain consistency, let's create a config file in the api directory that can store prefixes/constants for reuse across the workspace. This way, we can easily update the service name and other constants in one central place if needed.